src/components/PrimarySideBar.test.tsx

```diff
@@ -2,26 +2,26 @@ import { render, screen, userEvent } from '../utils/test-utils';
 import { PrimarySideBar } from './PrimarySideBar';
 
 describe('PrimarySideBar', () => {
-  it('should display Explorer and References icons', () => {
+  test('should display Explorer and References icons', () => {
```

Comment:
I think I left a comment about this elsewhere, but when we write "should" in a test description we should use `it` rather than `test` to make it read more clearly (I pretty much always use `it`) https://stackoverflow.com/questions/45778192/what-is-the-difference-between-it-and-test-in-jest